nix/treefmt.nix

```diff
@@ -0,0 +1,19 @@
+{
+  perSystem =
+    { pkgs, ... }:
+    {
+      treefmt = {
+        programs.nixfmt = {
+          enable = pkgs.lib.meta.availableOn pkgs.stdenv.buildPlatform pkgs.nixfmt-rfc-style.compiler;
```

Comment:
`nixfmt-rfc-style` is now just `nixfmt` and the previous name will be depecated, if I am not mistaken.